apps/laboratory/tests/shared/utils/project.ts

```diff
@@ -9,16 +11,35 @@ const PERMUTATIONS = availableDevices.flatMap(device =>
   LIBRARIES.map(library => ({ device, library }))
 )
 
+interface UseOptions {
+  launchOptions: {
+    executablePath: string
+  }
+}
+
 interface CustomProperties {
   testIgnore?: string
   testMatch?: string
+  useOptions?: UseOptions
 }
 
 export type CustomProjectProperties = {
   [T in string]: CustomProperties
 }
 
+const braveOptions: UseOptions = {
+  launchOptions: {
+    executablePath: getValue(BRAVE_UBUNTU_PATH, BRAVE_MACOS_PATH)
```

Comment:
won't this break locally for windows devs? not sure we have any though but jic

Reply:
It will, but we are fine with that for now